rotkehlchen/rotkehlchen.py

```diff
@@ -537,6 +537,20 @@ def unlock_user(
                     'Please contact rotki support via our discord to resolve this issue.',
                 )
 
+        # Send a notification to the user if data associated with
+        # old erc721 tokens has been saved during the db upgrade.
+        # TODO: Remove this after a couple versions (added in version 1.40).
+        with self.data.db.conn.read_ctx() as cursor:
+            if cursor.execute("SELECT COUNT(*) FROM sqlite_master WHERE type='table' AND name='user_added_solana_tokens'").fetchone()[0] != 0:  # noqa: E501
+                self.msg_aggregator.add_message(
+                    message_type=WSMessageType.SOLANA_TOKENS_MIGRATION,
+                    data={'identifiers': [i[0] for i in cursor.execute('SELECT identifier FROM user_added_solana_tokens')]},  # noqa: E501
+                )
+                self.msg_aggregator.add_warning(
+                    'Data associated with invalid ERC721 assets is present in your database. '
```

Comment:
The error message is also copy pasta. 

And if you do that then that means you can abstract it into a function and use it in both places